ocaml/idl/datamodel.ml

```diff
@@ -1499,6 +1499,8 @@ let _ =
     ~doc:"The USB group does not contain any PUSBs." ();
   error Api_errors.too_many_vusbs [ "number" ]
     ~doc:"The VM has too many VUSBs." ();
+  error Api_errors.usb_group_conflict [ "USB_group" ]
+    ~doc:"The USB_group must contain no vusb when creating VUSB object as now USB_group only have one PUSB." ();
```

Comment:
"USB_groups are currently restricted to contain no more than one VUSB"

Reply:
updated.